spec/index.html

```diff
@@ -1619,6 +1680,38 @@ <h2>Entailment rules (Informative)</h2>
       <p>As noted earlier, detecting datatype entailment for larger sets of datatype IRIs
         requires attention to idiosyncratic properties of the particular datatypes.
       </p>
+      
+      <table style="background-color: Cyan;color: Black">
+      <caption style="background-color: Cyan;color: Black">G-RDF entailment patterns with triple terms.</caption>
+        <tbody style="background-color: Cyan;color: Black">
+          <tr style="background-color: Cyan;color: Black">
+            <th > </th>
+            <th ><strong>if <em>S</em> contains</strong></th>
+            <th ><strong>then <em>S</em> RDF entails</strong></th>
+          </tr>
+          <tr >
+            <td class="othertable" style="background-color: Cyan;color: Black"><dfn>G-rdfREIF1</dfn></td>
+            <td class="othertable" style="background-color: Cyan;color: Black">sss aaa ttt <code>.</code><br/> ttt <code>rdf:type</code> <code>rdf:TripleTerm .</code><br/>for aaa ≠ <code>rdf:type</code></td>
+            <td class="othertable" style="background-color: Cyan;color: Black">aaa <code>rdf:type</code> <code>rdf:ReificationProperty .</code></td>
+          </tr>
+          <tr>
+            <td class="othertable" style="background-color: Cyan;color: Black"><dfn>G-rdfREIF2</dfn></td>
+            <td class="othertable" style="background-color: Cyan;color: Black">ttt aaa ooo <code>.</code><br/> ttt <code>rdf:type</code> <code>rdf:TripleTerm .</code><br/>for aaa ≠ <code>rdf:type</code></td>
+            <td class="othertable" style="background-color: Cyan;color: Black">aaa <code>rdf:type</code> <code>rdf:InverseReificationProperty .</code></td>
+          </tr>
+          <tr>
+          <tr>
+            <td class="othertable" style="background-color: Cyan;color: Black"><dfn>G-rdfTT1</dfn></td>
+            <td class="othertable" style="background-color: Cyan;color: Black">sss aaa ttt <code>.</code><br/> for ttt a triple term</td>
+            <td class="othertable" style="background-color: Cyan;color: Black">ttt <code>rdf:type</code> <code>rdf:TripleTerm .</code></td>
+          </tr>
```

Comment:
```suggestion
          </tr>
          <tr>
```

This is where the opening row should be.

Reply:
fixed